ci-operator/config/openshift-kni/scheduler-plugins/openshift-kni-scheduler-plugins-master.yaml

```diff
@@ -22,12 +22,12 @@ promotion:
 releases:
   initial:
     integration:
-      name: "4.11"
+      name: "4.12"
```

Comment:
if memory serves, back in time we used 4.11 here instead of the more correct 4.12 (so we used version N instead of the more correct version N+1) because version 4.12 (version N+1) was not available.
Let's merge this PR as-is, but let's post a followup to use 4.13 here. in the near future.

Reply:
Ack.